wgsl/diagrams/operators_1.mmd

```diff
@@ -0,0 +1,61 @@
+flowchart TD
+    classDef ltrGraph fill:#797;
+    classDef rtlGraph fill:#779,stroke-width:4px;
+    classDef noneGraph fill:#977,stroke-width:4px,stroke-dasharray: 5 5;
+    subgraph legend["Legend"]
+    legend0["Left-to-right"]:::ltrGraph
+    legend1("Right-to-left"):::rtlGraph
+    legend2["Requires parentheses"]:::noneGraph
+    end
+    A["(...)"]
+    subgraph primary[" "]
+    primary0["a()"]
+    primary1["a[]"]
+    primary2["a.b"]
+    end
+    subgraph unary[" "]
+    unary0("-a")
+    unary1("!a")
+    unary2("~a")
+    unary3("*a")
+    unary4("&a")
+    end
+    subgraph multiplicative[" "]
+    multiplicative0["a*b"]
+    multiplicative1["a/b"]
+    multiplicative2["a%b"]
+    end
+    subgraph additive[" "]
+    additive0["a+b"]
+    additive1["a-b"]
+    end
+    subgraph shift[" "]
+    shift0["a<<b"]
+    shift1["a>>b"]
+    end
+    subgraph relational[" "]
+    relational0["a<b"]
```

Comment:
Similar problem to the above, renders as `a`.

Reply:
Oops, sorry for the overlook! Tried to fix!